app/javascript/controllers/textarea_autogrow_controller.js

```diff
@@ -2,41 +2,28 @@ import { Controller } from '@hotwired/stimulus'
 
 export default class extends Controller {
 
-  initialize () {
+  initialize() {
     this.onResize = undefined
-    this.resizeDebounceDelayValue = 100
     this.autogrow = this.autogrow.bind(this)
   }
 
-  connect () {
+  connect() {
     this.element.style.overflow = 'hidden'
-    this.onResize = this.resizeDebounceDelayValue > 0 ? debounce(this.autogrow, this.resizeDebounceDelayValue) : this.autogrow
-
     this.autogrow()
 
     this.element.addEventListener('input', () => this.autogrow())
-    window.addEventListener('resize', () => this.onResize())
+    window.addEventListener('resize', () => this.debouncedAutogrow())
   }
 
-  disconnect () {
+  disconnect() {
     this.element.removeEventListener('input', () => this.autogrow())
-    window.removeEventListener('resize', () => this.onResize())
+    window.removeEventListener('resize', () => this.debouncedAutogrow())
   }
 
-  autogrow () {
+  debouncedAutogrow = _.debounce(this.autogrow, 100)
+
+  autogrow() {
     this.element.style.height = 'auto'
     this.element.style.height = `${this.element.scrollHeight + 2}px` // the +2 is a hack to make the size not jump on load. The scrollHeight differs from than actual height for the empty state
   }
-}
-
-function debounce (callback, delay) {
-  return (...args) => {
-    const context = this
-    if (typeof timeout !== 'undefined')
-      clearTimeout(timeout)
-    else
-      var timeout = undefined
-
-    timeout = setTimeout(() => callback.apply(context, args), delay)
-  }
```

Comment:
Since a second time has arisen for a debouncer I factored it out into the global utility.